mobile/lib/main.dart

```diff
@@ -43,16 +44,16 @@ class MainApp extends HookWidget {
   }
 }
 
-// TODO: needs major refactor
+// TODO: decide if needs further refactor
```

Comment:
I'd say it needs moving to a separate file, like all the other pages in our app. I'd add this to the TODO message, but that's my personal view.